app/src/main/java/com/bunbeauty/papakarlo/feature/consumer_cart/ConsumerCartFragment.kt

```diff
@@ -116,55 +148,62 @@ class ConsumerCartFragment : BaseFragment(R.layout.fragment_consumer_cart) {
                             },
                             onCountDecreased = {
-                                viewModel.removeProductFromCart(cartProductItemModel.menuProductUuid)
+                                removeProductFromCartClicked.invoke(cartProductItemModel.menuProductUuid)
+                            },
+                            onClick = {
+                                onProductClicked.invoke(cartProductItemModel)
                             }
-                        ) {
-                            viewModel.onProductClicked(cartProductItemModel)
-                        }
+                        )
                     }
                 }
-                BlurLine(modifier = Modifier.align(Alignment.BottomCenter))
             }
-            Column(
+            Surface(
                 modifier = Modifier
+                    .fillMaxWidth()
                     .background(FoodDeliveryTheme.colors.mainColors.surface)
-                    .padding(FoodDeliveryTheme.dimensions.mediumSpace)
+                    .zIndex(1f),
+                shadowElevation = 6.dp,
+                color = FoodDeliveryTheme.colors.mainColors.surface
             ) {
-                Row {
-                    Text(
-                        text = stringResource(R.string.title_consumer_cart_total),
-                        style = FoodDeliveryTheme.typography.h2,
-                        color = FoodDeliveryTheme.colors.mainColors.onSurface
-                    )
-                    Row(
-                        modifier = Modifier.weight(1f),
-                        horizontalArrangement = Arrangement.End
-                    ) {
-                        consumerCart.oldTotalCost?.let { oldTotalCost ->
-                            Text(
-                                modifier = Modifier
-                                    .padding(end = FoodDeliveryTheme.dimensions.smallSpace),
-                                text = oldTotalCost,
-                                style = FoodDeliveryTheme.typography.h2,
-                                color = FoodDeliveryTheme.colors.mainColors.onSurfaceVariant,
-                                textDecoration = TextDecoration.LineThrough
-                            )
-                        }
+                Column(
+                    modifier = Modifier
+                        .padding(FoodDeliveryTheme.dimensions.mediumSpace)
+                ) {
+                    Row {
                         Text(
-                            text = consumerCart.newTotalCost,
-                            style = FoodDeliveryTheme.typography.h2,
+                            text = stringResource(R.string.title_consumer_cart_total),
+                            style = FoodDeliveryTheme.typography.bodyMedium.bold,
                             color = FoodDeliveryTheme.colors.mainColors.onSurface
                         )
+                        Row(
```

Comment:
Let's use in such cases `Spacer(modifier = Modifier.weight(1f))`. It's shorter and less nesting.